frontend/src/app/map/map.component.html

```diff
@@ -0,0 +1,3316 @@
+    <polyline class="blackThickerInnerWallsClass"
+      points="1953.5,959.5 1953.5,578.5 1763.5,578.5 1763.5,605.5 1791.5,605.5 	" />
+    <polyline class="blackThickerInnerWallsClass"
+      points="1813.5,405.5 1693.5,405.5 1693.5,533.5 1801.5,533.5 1801.5,442.5 	" />
+    <line class="blackThickerInnerWallsClass" x1="1695.5" y1="492.5" x2="1801.5" y2="492.5" />
+    <polyline class="blackThickerInnerWallsClass" points="1843.5,481.5 1843.5,405.5 1906.5,405.5 1906.5,547.5 	" />
+    <polyline class="blackThickerInnerWallsClass" points="1843.5,506.5 1843.5,533.5 1906.5,533.5 	" />
+    <line class="blackThickerInnerWallsClass" x1="1801.5" y1="418.5" x2="1801.5" y2="405.5" />
+    <line class="blackThickerInnerWallsClass" x1="1815.5" y1="605.5" x2="1827.5" y2="605.5" />
+    <line class="blackThickerInnerWallsClass" x1="1827.5" y1="608.5" x2="1827.5" y2="578.5" />
+    <polyline class="blackThickerInnerWallsClass" points="1876.3,578.5 1876.3,605.5 1952.5,605.5 	" />
+    <polyline class="blackThickerInnerWallsClass" points="491.5,605.5 479.5,605.5 479.5,579.5 	" />
+    <polyline class="blackThickerInnerWallsClass" points="429.7,579.5 429.7,605.5 354.5,605.5 	" />
+    <line class="blackThickerInnerWallsClass" x1="422.5" y1="991.5" x2="422.5" y2="1111.5" />
+    <line class="blackThickerInnerWallsClass" x1="461.5" y1="492" x2="355" y2="492" />
+    <line class="blackThickerInnerWallsClass" x1="504.5" y1="492" x2="613.5" y2="492" />
+    <line class="blackThickerInnerWallsClass" x1="421.5" y1="885.5" x2="421.5" y2="898.5" />
+    <line class="blackThickerInnerWallsClass" x1="421.5" y1="925.5" x2="421.5" y2="946" />
+    <line class="blackThickerInnerWallsClass" x1="1884.2" y1="885.5" x2="1884.2" y2="897.5" />
+    <line class="blackThickerInnerWallsClass" x1="1884.5" y1="925.5" x2="1884.5" y2="945.5" />
+    <line class="blackThickerInnerWallsClass" x1="437.5" y1="644.5" x2="437.5" y2="844.5" />
+    <line class="blackThickerInnerWallsClass" x1="1868.5" y1="644.5" x2="1868.5" y2="840.5" />
+    <line class="blackThickerInnerWallsClass" x1="1693.4" y1="240" x2="1693.4" y2="350" />
+    <line class="blackThickerInnerWallsClass" x1="1694" y1="240" x2="1694" y2="350" />
+    <line class="blackThickerInnerWallsClass" x1="2021.6" y1="540" x2="2021.6" y2="580" />
+
+  </g>
+
+  <g id="black-outer-walls">
+    <polygon class="blackOuterWallsClass" points="25.5,22.5 807.5,22.5 807.5,915.5 1499.5,915.5 1499.5,214.5 1954.5,214.5 1954.5,533.5
+   2188.5,533.5 2188.5,1349.5 119.5,1349.5 	" />
+  </g>
+
+  <g id="white-inner-walls">
+    <polyline class="whiteInnerWallsClass" points="38.8,210.7 221.5,210.7 221.5,168.5 	" />
+    <line class="whiteInnerWallsClass" x1="221.5" y1="127.5" x2="221.5" y2="22.5" />
+    <line class="whiteInnerWallsClass" x1="285.5" y1="127.5" x2="285.5" y2="22.5" />
+    <polyline class="whiteInnerWallsClass" points="43.3,274.4 221.5,274.4 221.5,314.5 	" />
+    <line class="whiteI
```

Comment:
Pisałem o tym w review do prototypów
"Paragraf "Avoiding Element Selectors For Components":https://teropa.info/blog/2016/12/12/graphics-in-angular-2.html"